src/app/code/community/EasyTranslate/Connector/Model/Resource/Project.php

```diff
@@ -290,4 +290,14 @@ protected function _lookupTargetStoreIds(int $id): array
 
         return $adapter->fetchCol($select, $binds);
     }
+
+    private function randomiseString()
+    {
+        $strong_result = true;
+        if (function_exists('random_bytes')) {
```

Comment:
Please add a comment that this is only available in PHP >= 7 and that we fall back to the openssl version for PHP 5 support.